robot/impl/resource_manager.go

```diff
@@ -819,50 +819,63 @@ func (manager *resourceManager) completeConfig(
 }
 
 func (manager *resourceManager) completeConfigForRemotes(ctx context.Context, lr *localRobot) {
+	// Add remotes in parallel. This is particularly useful in cases where
+	// there are many remotes that are offline or slow to start up.
+	var remoteErrGroup errgroup.Group
+	remoteErrGroup.SetLimit(5)
 	for _, resName := range manager.resources.FindNodesByAPI(client.RemoteAPI) {
 		gNode, ok := manager.resources.Node(resName)
 		if !ok || !gNode.NeedsReconfigure() {
-			continue
-		}
-		var verb string
-		if gNode.IsUninitialized() {
-			verb = "configuring"
-		} else {
-			verb = "reconfiguring"
+			return
```

Comment:
I see you introduced a lambda on the line below. If this is truly intended to be a return, was the intention to add this to that lambda?